templates/home.html

```diff
@@ -142,7 +142,9 @@ <h4 class="unlockFont">Upload ICS file of your timetable from <a href="https://g
 					<input id="file-in" type="file" class="form-control col-auto" onchange="readICS()">
 				</div>
 				<div id="details-div"></div>
-
+				<p id=".footer">
+				<strong>Contribute to this project on <a href="https://github.com/metakgp/what-slot"> GitHub </a> | Powered by <a href="https://metakgp.github.io/">Metakgp</a> with ♥ </strong>
```

Comment:
Please remove the spaces around "Github". My apologies that I missed it in the first review.